generate_client_cert.sh

```diff
@@ -35,6 +35,7 @@ ORGANISATION_NAME="$ORGANISATION_DOMAIN"                    # Organization Name
 ORGANISATION_UNIT="${ORGANISATION_UNIT:-}"                  # Organizational Unit Name
 COMMON_NAME="${COMMON_NAME:-$ORGANISATION_DOMAIN}"          # Common Name
 EMAIL_ADDRESS="${EMAIL_ADDRESS:-root@$ORGANISATION_DOMAIN}" # Email Address
+CERTIFICATE_TYPE="${CERTIFICATE_TYPE:-usr_cert}"
```

Comment:
If we go for `client_cert` over `usr_cert` then there will need to be a change here